.github/workflows/django.yml

```diff
@@ -10,22 +10,23 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    strategy:
-      max-parallel: 4
-      matrix:
-        python-version: [3.6, 3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python 3.8
       uses: actions/setup-python@v1
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: 3.8
     - name: Install Dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install pipenv
+        pip install prospector
+        pipenv install
+    - name: Run Lint
+      run: |
+        pipenv run prospector
```

Comment:
Check the implementation of runlint, it has errors in Django.yml not allowing the other tests to run

Reply:
Yes, those are the lint errors of the whole project. I do not intend to fix them to keep the pull request small (no breaking changes across many files). Keep in mind this is working with python 3.8

Reply:
So these errors seem to us due to vscode and Django? The project was developed with Python 3.8.3, no problem with that part

Reply:
Those are lint errors they come from the pep8 code style guide. Meaning that the base project is way far of the pep8 coding standard.

Reply:
So in this case i would have to change the base code  to comply with pep8

Reply:
Yep, but that's another story

Reply:
Master lint checked, should we merge? @leortyz